tsPractice/src/PlusOne/plusOne.ts

```diff
@@ -0,0 +1,33 @@
+export function plusOne(inputArray: number[]): number[] {
+    let modifiedArray: number[] = inputArray;
```

Comment:
This variable is still pointing to the same object reference as the parameter array.

So your changed to `modifiedArray` are also applying to the `inputArray`.

Might want to make an actual copy of the data into a new array so that the parameter is not changed.  (`array.slice()` might help here).

Reply:
Generally, a function that returns something new should avoid modifying the parameter objects.  And if the function is designed to modify the parameter objects correctly, usually we don't return it back out as a new variable.

This helps make it more obvious that the function will modify the parameter object "in place"